provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -37,11 +38,52 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
             nodes.append(self._create_node(node))
         return nodes
 
+    def _create_pool_descriptions(self) -> List[PoolDesc]:
+        pools: List[PoolDesc] = []
+        conf = self.provider
+        cluster_id = conf.get('cluster>cluster_id')
+        storage_set_count = int(
+            conf.get(f'cluster>{cluster_id}>site>storage_set_count'))
+
```

Comment:
before going on to build disk refs for a storage set lets validate if the #data_devices >= data_units + parity_units + spare_units. If this condition is not satisfied we must fail logging a failure message as "Invalid storage set configuration".

Reply:
Done